lighthouse-core/report/html/renderer/util.js

```diff
@@ -418,69 +418,67 @@ class Util {
 
     return [
       {
-        name: 'Device',
+        name: this.UIStrings.runtimeSettingsDevice,
```

Comment:
any reason we need to use `this` instead of `Util` here?

Reply:
No not really, I was following the prior art for that file, but using `this.` for only this file seems bad.